runtime/src/accounts_db.rs

```diff
@@ -4007,17 +4006,15 @@ impl AccountsDb {
             .is_none());
     }
 
-    pub fn create_drop_bank_callback(
-        &self,
-        pruned_banks_sender: DroppedSlotsSender,
-    ) -> SendDroppedBankCallback {
+    /// Set the bank-drop-callback flag to ENABLED
+    pub(crate) fn enable_bank_drop_callback(&self) {
         self.is_bank_drop_callback_enabled
             .store(true, Ordering::SeqCst);
```

Comment:
Also, I don't see why this atomic needs Sequential Consistency for its ordering. From what I can tell, this `.store()` could use `Release`, and the `.load()` below could use `Acquire`.

I wanted to keep this PR as simple as possible, so exploring different orderings is left for a subsequent PR.